resources/styles/components/performance.less

```diff
@@ -0,0 +1,18 @@
+.performance-report {
+  table {
+    th {
+      .no-select();
+      &.sortable {
+        cursor: pointer;
+      }
+      &.asc:after {
+        content: '\00a0\f0de';
+        font-family: FontAwesome;
+      }
+      &.desc:after {
```

Comment:
Can you use `ascending` and `descending` or `is-ascending` and `is-descending`? `desc` could also be a `description`